spec/features/projects/projects_index_spec.rb

```diff
@@ -124,14 +140,14 @@ def remove_filter(name)
   feature 'restricts project visibility' do
     feature 'for a anonymous user' do
       scenario 'only public projects shall be visible' do
+        Role.anonymous
         visit projects_path
 
         expect(page).to_not have_text(project.name)
         expect(page).to have_text(public_project.name)
 
         # Test that the 'More' menu stays invisible on hover
-        page.find('tbody tr').hover
-        expect(page).to_not have_css('.icon-show-more-horizontal')
+        expect(page).to_not have_selector('.icon-show-more-horizontal')
```

Comment:
Hmm, is this still the same? Don't we need to ensure that we hovered first so that we can be assured that the menu does not show up then??